tools/language-server/wstest/set-expression-value/input.txt

```diff
@@ -0,0 +1,10 @@
+{ "jsonrpc": "2.0", "method": "text/applyExpressionValue", "id": 0, "params": { "expressionId": "524dd815-b652-4bbe-b9f2-26b35d17993a", "expressionValue":"9", "edit": { "path":{"rootId":"6f7d58dd-8ee8-44cf-9ab7-9f0454033641","segments":["src","Main.enso"]}, "edits":[{"range":{"start":{"line":4,"character":15},"end":{"line":4,"character":16}}, "text":"9"}], "oldVersion":"e980e79db6f43cfb85f74b642a5df42b46ff6827ae191e2551d82456", "newVersion":"6dea845964972ebdceccc36e3d4605324fc54bde40709e5d35c1a93e"} } }
```

Comment:
Usual benchmarks running with [JMH](https://openjdk.org/projects/code-tools/jmh/) specify:
- _operation to perform_ 
- number of repetitions to perform to _warm up_
- number of _iterations to measure_

This file seems to explicitly specify five repetitions of two operations. Rather than that, it should specify just the two messages to send and them _warm up_ times without logging and _iterations to measure_ times to. And somehow use just the _measure iterations_ to compute the result then.